AllReadyApp/Web-App/AllReady.UnitTest/Controllers/AccountControllerTests.cs

```diff
@@ -415,74 +468,183 @@ public void RegisterPostHasValidateAntiForgeryTokenAttribute()
             Assert.NotNull(attribute);
         }
 
-        [Fact(Skip = "NotImplemented")]
+        [Fact]
         public async Task LogOffInvokesSignOutAsync()
         {
-            //delete this line when starting work on this unit test
-            await taskFromResultZero;
+            var userManager = CreateUserManagerMock();
+            var signInManager = CreateSignInManagerMock(userManager);
+
+            var sut = new AccountController(userManager.Object, signInManager.Object, null,null, null);
+
+            await sut.LogOff();
+            signInManager.Verify(x => x.SignOutAsync(), Times.Once);
         }
 
-        [Fact(Skip = "NotImplemented")]
+        [Fact]
         public async Task LogOffRedirectToCorrectActionAndController()
         {
-            //delete this line when starting work on this unit test
-            await taskFromResultZero;
+            var userManager = CreateUserManagerMock();
+            var signInManager = CreateSignInManagerMock(userManager);
```

Comment:
you can change this line to: `var signInManager = CreateSignInManagerMock(CreateUserManagerMock());`